packages/authjs-nuxt/src/runtime/composables/useAuth.ts

```diff
@@ -50,6 +50,10 @@ export function useAuth() {
     }
   }
 
+  const getProviders: () => ReturnType<typeof auth.getProviders> = () => {
```

Comment:
type can be inferred here

Reply:
Cool cool. I removed the type here. It should be inferred.